src/javascript/_common/base/client_base.js

```diff
@@ -82,6 +82,8 @@ const ClientBase = (() => {
         return account_type;
     };
 
+    const getBrokerCode = loginid => loginid.match(/[A-Z]*/)[0];
+
```

Comment:
I think we have a function called `getAccountType()` I think it kind of does the same as what you do here, but instead translates the broker codes to account types. Perhaps, you can use this function without writing a new one?

Reply:
Thanks for suggestion, Hubert! It's more right way to do a check with `getAccountType()`. I don't understand meaning of account types in the first time. Removed.